Automattic/wp-calypso
# Add Term Toggle for Jetpack Products in Cart

#### Changes proposed in this Pull Request

* Modify variant item handlers to also handle Jetpack Products
<img width="931" alt="Screen Shot 2021-10-14 at 11 33 11 AM" src="https://user-images.githubusercontent.com/2810519/137375749-4370b931-61d2-4b8d-bc75-8e4e92c0e981.png">

#### Testing instructions
1. Boot branch as Calypso Blue
2. Add Jetpack Daily Backup to Cart
3. Verify that when you click "Edit" in the cart you can change the term of the Jetpack Product
4. Repeat steps 2 & 3 for a WordPress.com plan that currently supports term switching ( any of personal, professional, etc should work ) and make sure it still works.
